src/CloudFoundry.CloudController.Test.Integration/PushTest.cs

```diff
@@ -63,7 +63,7 @@ public static void ClassInit(TestContext context)
 
             if (winStack == Guid.Empty)
             {
-                throw new Exception("Could not test on a deployment without a windows 2012 stack");
+                Assert.Inconclusive("Could not test on a deployment without a windows 2012 stack");
```

Comment:
We've had issues with inconclusive - I remember using categories to decide which tests we want to run against a certain target?